pkg/process/config/config_test.go

```diff
@@ -290,7 +291,7 @@ func TestAgentConfigYamlAndSystemProbeConfig(t *testing.T) {
 	assert.Equal(1000, agentConfig.ClosedChannelSize)
 	assert.Equal(agentConfig.ExcludedBPFLinuxVersions, []string{"5.5.0", "4.2.1"})
 	assert.Equal("/var/my-location/system-probe.log", agentConfig.SystemProbeAddress)
-	assert.Equal(append(processChecks, "connections"), agentConfig.EnabledChecks)
+	assert.Equal(append(processChecks), agentConfig.EnabledChecks)
```

Comment:
The contents of `/testdata/TestDDAgentConfigYamlAndSystemProbeConfig-Net-2.yaml` are

```
system_probe_config:
    enabled: true
    sysprobe_socket: /var/my-location/system-probe.log
    disable_tcp: true
    disable_udp: true
    disable_ipv6: true
    disable_dns_inspection: false
    excluded_linux_versions:
      - 5.5.0
      - 4.2.1
    closed_channel_size: 1000
    source_excludes:
      127.0.0.1:
        - "5005"
      172.0.0.1/20:
        - "*"
      "*":
        - "443"
    dest_excludes:
      2001:db8::2:1:
        - "5005"
      172.0.0.1/20:
        - "*"
      "*":
        - "*"

network_config:
    enabled: false
```